### ISSUE TITLE
Images in Assets/Images Directory Not Displaying in Build or Development Mode

### ISSUE THREAD

**Issue by joannerpena** [cearls/timberland#46]
#### Description

When attempting to reference images uploaded to the `Assets/Images` directory, the images do not display either during development (`npm run dev`) or after building the project. 

#### Error Messages

The terminal output shows the following errors:

```shell
../images/caution_icon.svg referenced in ../images/caution_icon.svg didn't resolve at build time, it will remain unchanged to be resolved at runtime

/wp-content/themes/timberland/theme/assets/images/down_chevron.svg referenced in /wp-content/themes/timberland/theme/assets/images/down_chevron.svg didn't resolve at build time, it will remain unchanged to be resolved at runtime
```

Additionally, the URL generated for the images, such as `https://localhost:3000/theme/assets/images/down_chevron.svg`, does not display the images correctly.

#### Environment

- **Tooling:** Using Vite with the default configuration provided by the project.

#### Steps to Reproduce

1. Place an image file (e.g., `caution_icon.svg`) in the `Assets/Images` directory.
2. Reference the image in the project.
3. Run `npm run dev` or build the project.
4. Observe the errors in the terminal and the broken image URL in the browser.

#### Expected Behavior

The images should resolve correctly and be displayed both during development and in the built version of the project.

#### Actual Behavior

- Images do not display.
- Terminal outputs errors stating the references didn't resolve at build time.
- Generated URLs for images are incorrect or inaccessible.

#### Additional Information

The issue persists with the default Vite configuration.

#### Question

Am I missing something in the configuration, or is there a specific step needed to ensure that images in the `Assets/Images` directory resolve correctly? Any suggestions or guidance would be appreciated.

**Comment by ineptian** [comment]
@joannerpena did you ever figure this out? I am having this issue as well with both images and fonts. Would love to get it figured out.

**Comment by bdfpplinsinga** [comment]
Can i ask you how you reference the image inside your theme? I tested it and it works. But you do need to reference it by pointing to the theme folder:

`<img src="{{ theme.uri }}/assets/images/chevron-down.svg" alt="">`

**Comment by ineptian** [comment]
@bdfpplinsinga Yeah I was able to figure out the theme images. I am still not certain how I would reference fonts from a css file though since I dont believe I can use `{{ theme.uri }}` there

**Comment by joannerpena** [comment]
@bdfpplinsinga @ineptian I figured out the using images inside my Twig templates but the use of images on CSS or JS is still an issue I haven't been able to fix.
